cp2k_output_tools/blocks/energies.py

```diff
@@ -13,9 +30,16 @@
 
 
 def match_energies(content: str) -> Optional[BlockMatch]:
+    spans = []
+    energies = {}
+    match = MAIN_ENERGY_RE.search(content)
+    if match:
+        energies = match.groupdict()
+        spans = match.spans(0)
```

Comment:
```suggestion
        spans = [match.spans(0)]
```

`mypy` should have complained when initialising `BlockMatch` further down below.
According to signature the spans should be a `List[Union[int, Tuple[int, int]]]`, but `[] += (1,2)` becomes `[1, 2]`, not `[(1, 2)]`.